safe-global/safe-wallet-web
# fix: Apply input styles to all input fields

## What it solves

Part of #2067 

## How this PR fixes it

- Extracts inputs styles into a common css file `inputs.module.css`
- Applies the new input styles (66px height, border-color) to all input fields

## How to test it

1. Open a Safe
2. Create/Add a safe and observe all input fields have the same border color and height
3. Do the same for all other flows
4. Check the transaction history filter

## Screenshots
<img width="797" alt="Screenshot 2023-07-05 at 11 45 27" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/9e4318ad-1c3c-4f3f-986b-220dd112e716">
<img width="1266" alt="Screenshot 2023-07-05 at 11 45 58" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/5488b673-dadd-4fe1-a89e-56af50e57bc1">

## Checklist
